Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -443,7 +450,8 @@ - (void)updateTableContents
         statusLabel.text = muteStatus;
         [cell.contentView addSubview:statusLabel];
         [statusLabel autoVCenterInSuperview];
-        [statusLabel autoPinEdgeToSuperviewEdge:ALEdgeRight];
+        //        [statusLabel autoPinLeadingToTrailingOfView:rowLabel margin:weakSelf.iconSpacing];
```

Comment:
Removed. And I recently learned that there's a better way to do this (have about an accessory type _AND_ an accessory label.  You use `UITableViewCellStyleValue11 and `detailTextLabel`.